db/migrate/20220123185631_add_initial_user_information.rb

```diff
@@ -1,10 +1,9 @@
 class AddInitialUserInformation < ActiveRecord::Migration[7.0]
   def change
     add_column :users, :name, :string
-    add_column :users, :role, :integer
+    add_column :users, :role, :string
```

Comment:
Why are you modifying a migration? 🤔

Reply:
user role must be string, I'm not sure why role was integer but that is an error

Reply:
I'm solve this